src/screens.d.ts

```diff
@@ -3,13 +3,13 @@
 
 declare module 'react-native-screens' {
   import { ComponentClass } from 'react';
-  import { ViewProps } from 'react-native';
+  import { ViewProps, Animated } from 'react-native';
 
   export function useScreens(shouldUseScreens?: boolean): void;
   export function screensEnabled(): boolean;
 
   export interface ScreenProps extends ViewProps {
-    active?: boolean;
+    active?: 0 | 1 | Animated.AnimatedInterpolation;
```

Comment:
why? Shouldn't it be just `0 | 1`?

Reply:
We're using an animated value in React Navigation. It's also an `Animated.View` so it should support animated nodes: https://github.com/kmagiera/react-native-screens/blob/0d082ad98566925366e35b6f6835c6dfa3d33cd5/src/screens.native.js#L61

Reply:
only style properties are supported by reanimated.

Reply:
@osdnk The type is for vanilla Animated :)

Reply:
only style properties are supported by Animated as well

Reply:
@osdnk It can support any prop. Check this out https://snack.expo.io/SJLybJP2N

Reply:
This line https://github.com/facebook/react-native/blob/ddce74388a5662b5307582ca2e05710224c74ba0/Libraries/Animated/src/createAnimatedComponent.js#L149

Reply:
But it still won't be supported by native drivers, am I right?

Reply:
Yea, it won't be supported by native driver